templates/taxi/index.html

```diff
@@ -10,4 +10,6 @@ <h2>Dynamic content</h2>
     <li><strong>Drivers:</strong> {{ num_drivers }}</li>
     <li><strong>Manufacturers:</strong> {{ num_manufacturers }}</li>
   </ul>
+  <p>You have visited this page {{ num_visits }}
+      {% if num_visits == 1 %} time {% else %} times {% endif %} </p>
```

Comment:
According to checklist, better to use pluralize. Good example:
`<p>You have visited this page {{ num_visits }} time{{ num_visits|pluralize }}</p>`